### ISSUE TITLE
Blcok a ticket due to an external need/dependency

### ISSUE THREAD

**Issue by gmdesousa** [kanboard/kanboard#516]
Sometimes in our team we have a situation that a ticket is blocked due to a external need. 
We keep it on the "doing" stage but on a "on hold" or "blocked" state so we can remember to verify if the external dependency is resolved.
It's really nice, but we can't use a column to simulate this as we are not allowed to move tickets backward.
The tickets that are blocket don't count on our WIP so we can pick another ticket.
It would be nice to have this feature implemented on Kanboard.

**Comment by adilbo** [comment]
Hope i understand what is your need, but why dont you use a category e.g. STOP with an extra color for this?

**Comment by David-Norris** [comment]
Perhaps a blocked or on hold swimlane? The ticket could then stay in the same column (but would be on a different row), and also wouldn't count towards any limits for your active tickets.

**Comment by gmdesousa** [comment]
The idea of a column is interesting and we are using it now.
The problem is that we feel guilty for move the ticket backward to the "running" column, as many kanban user tell us to never do it.
Anyway, do you agree that this feature is important so Kanboard could be more compliant to Kanban?

**Comment by benzvan** [comment]
This is an important feature to me. While you can manually mark a card as being blocked/unblocked, it would be more seamless if the card would come unblocked when the blocking cards are moved to done/closed rather than having to look at the cards constantly to find out if the blockers are still blocking.

**Comment by bugwatcher** [comment]
This issue has been closed automatically for inactivity after more than 1 year. If you still need this feature, you should consider contributing to this open-source project.
